### ISSUE TITLE
Favorites

### ISSUE THREAD

**Issue by chesterbr** [chesterbr/toronto-transit-time#6]
It has been suggested that we could allow saving routes as favorites

(possibly for when you are within a threshold of their stop? That may be something to do once the options screen - #3 - is implemented)

**Comment by bwinton** [comment]
There are a _lot_ of stops near my office, but I'm only interested in one of them.  Just pushing that one to the top would make the app _much_ more useful!  😃

Also, you don't need the options screen for this.  You could have the Select button toggle a favourite status on the stop details screen…

**Comment by chesterbr** [comment]
Agreed. I think a long-press for options (or even activating that once you enter the stop) could store the favorite. Thanks for the tip!

**Comment by bwinton** [comment]
My pleasure!  I might be starting to code the select button version over [here](https://github.com/bwinton/toronto-transit-time/commit/f0a59c138db323e62093b567f0f84c73a5ac963d), as a way of getting back into Pebble development, so if you don't feel like implementing this, a pull request will probably be coming your way eventually…  😉

**Comment by chesterbr** [comment]
Sure, please go ahead! I'll won't be able to jump into coding new features in the upcoming weeks, so PRs are welcome (and will get prompt feedback / testing / merging / publishing, don't worry).
